### PR TITLE
Update istioctl auth tls-check example to reflect https://github.com/istio/istio/pull/17720

### PR DESCRIPTION
**Pull request by diemtvu**
Please provide a description for what this PR is for.

https://github.com/istio/istio/pull/17720 swap the name & namespace order in the output.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[X] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/tasks/security/mutual-tls/index.md [hunk 1]
```diff
@@ -103,9 +103,9 @@ The output shows:
 
 * `CLIENT`: the mode used on the client or clients.
 
-* `AUTHN POLICY`: the name and namespace of the authentication policy. If the policy is the mesh-wide policy, namespace is blank, as in this case: `default/`
+* `AUTHN POLICY`: the namespace and name of the authentication policy. If the policy is the mesh-wide policy, namespace is blank, as in this case: `/default`
```
Comment (frankbu) [inline content/en/docs/tasks/security/mutual-tls/index.md]: Why is the authentication policy syntax and destination rule syntax not consistent? AP is `namespace/x` while DR is `x/namespace`. That inconsistency seems unnecessary and confusing.

**Comment by frankbu** [comment]
Does this FAQ entry also need to be updated: https://preliminary.istio.io/faq/security/#check-policy

File: content/en/docs/tasks/security/mutual-tls/index.md (hunk 1 above)
Comment (diemtvu) [reply content/en/docs/tasks/security/mutual-tls/index.md]: My bad, forgot to update the value for destination rule. Both are namespace/name order.

**Comment by frankbu** [comment]
@diemtvu Please update https://preliminary.istio.io/faq/security/#check-policy also.

**Comment by diemtvu** [comment]
> @diemtvu Please update https://preliminary.istio.io/faq/security/#check-policy also.

Done

**Pull request merged**